erpnext/public/js/controllers/taxes_and_totals.js

```diff
@@ -391,9 +391,27 @@ erpnext.taxes_and_totals = erpnext.payments.extend({
 			? this.frm.doc["taxes"][tax_count - 1].total + flt(this.frm.doc.rounding_adjustment)
 			: this.frm.doc.net_total);
 
+		var paid_amount = 0.0;
+		var base_paid_amount = 0.0;
+		var base_grand_total = 0.0;
```

Comment:
This variable is used only once.
Let's use `me.frm.doc.base_grand_total` instead.

Reply:
Fixed.